lib/src/parser/ExampleGrammar.pest

```diff
@@ -0,0 +1,3 @@
+ID = {ASCII_ALPHA_UPPER~ASCII_ALPHA+}
```

Comment:
Is this still needed?

Reply:
Only to make the code compile right now. As soon as the parser team pushes their newest changes, we can remove this